roles/ckan/templates/ckan/ckan_production.ini

```diff
@@ -127,13 +127,15 @@ ckanext.emailasusername.require_user_email_input_confirmation = False
 ckan.plugins = {{ ckan_plugins }}
 
 
+{% if ckan_googleanalytics_enable %}
 ## Google analytics settings
-googleanalytics.measurement_id = G-MCS01BTG6Q
-googleanalytics.property_id = 448344994
+googleanalytics.measurement_id = {{ ckan_googleanalytics_measurement_id }}
+googleanalytics.property_id = {{ ckan_googleanalytics_property_id }}
 googleanalytics.account = who.afro.ckan
 googleanalytics.username = who.afro.ckan@gmail.com
 googleanalytics.show_downloads = true
 googleanalytics.download_handler = ckanext.blob_storage.blueprints:download
```

Comment:
Sorry I didn't spot this earlier Ntwali, but this all looks project specific to me? Not just measurement_id and property_id.  Especially account and username which actually includes the project name in the values. 

You could create seperate ansible variables for every line, but the more concise way (and perhaps more consistent way with how we are configuring other extensions) is to just expect the user to configure the extension through the `ckan_extras` variable printed out in line 243(245) of this production.ini template (e.g. see variable value [here](https://github.com/fjelltopp/fjelltopp-infrastructure/blob/1fbd1df93b80626f168ebef697e9e999f3303ad7/inventory/group_vars/wac_local_dev/settings.yml#L39-L51)).

Reply:
Only the `measurement_id` and `property_id` are project specific.
If we decide to use Google analytics in another project, only those two will need to change.
In fact, an oversight on my side, the `show_downloads` option is no longer required (after the refactoring).

Kindly allow me 10 mins to think carefully about how to use `ckan_extras`.

Reply:
It looks like `ckan_extras` is defined only in https://github.com/fjelltopp/fjelltopp-ansible/blob/master/roles/ckan/defaults/main.yml. Is this file meant for project-specific configurations? I thought the way currently done, where we specify project-specific data in the appropriate inventory group_vars was the way to go?